__tests__/app.test.js

```diff
@@ -191,8 +196,49 @@ describe('/api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('400 Bad Request: Invalid Id')
         })
     })
-})
+    test('POST 201: responds with the a named object containing the posted comment', () => {
+        const input = {
+            //Presumably the user would have to have an account before posting, so i used an already existing user
+            //Please let me know if this is correct in the feedback!
+            username: 'lurker',
+            body: 'Cool article!'
+        }
+
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(input)
+        .expect(201)
+        .then(({body}) => {
+            expect(body.comment).toEqual({author: 'lurker', body: 'Cool article!'})
+            expect(body.comment).toBeObject()
+        })
+    })
+    test('POST 400: responds with a 400 error when there are missing/malformed fields', () => {
+        const input = {}
+
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(input)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe('400 Bad Request: missing/malformed fields')
+        })
+    })
+    test('POST 400: responds with a 404 error when the article_id doesnt exist', () => {
+        const input = {
+            username: 'lurker',
+            body: 'Cool article!'
+        }
 
+        return request(app)
+        .post('/api/articles/99999/comments')
+        .send(input)
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("404 Error: Resource doesn't exist")
+        })
+    })
+})
 
```

Comment:
Please add another test where the username given in the input object doesn't exist in the database. (404).